met-api/migrations/versions/f2b3f08c8d60_alter_comment_add_submission_id.py

```diff
@@ -0,0 +1,52 @@
+"""alter comment add submission_id
+
+Revision ID: f2b3f08c8d60
+Revises: c19bc1af9f2b
+Create Date: 2022-11-01 11:59:02.563717
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f2b3f08c8d60'
+down_revision = 'c19bc1af9f2b'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    conn = op.get_bind()
+    op.add_column('comment', sa.Column('submission_id', sa.Integer(), nullable=True))
+    op.add_column('comment', sa.Column('component_id', sa.String(), nullable=True))
```

Comment:
Should we call component_id as question_id ? was wondering how to make it easy to convey that component_id is referring to the question thats being asked?

Reply:
since we are going through the list of components to fetch its ids that what I thought was the name of it, i don't think. 'question' works too because it could be labeled as something like 'Leave your comment' which technically is not a question either

Reply:
does survey_component_id sounds any better?